pkg/agent/openflow/client.go

```diff
@@ -580,6 +605,14 @@ func (c *client) InstallGatewayFlows() error {
 	if c.encapMode.SupportsEncap() {
 		flows = append(flows, c.l3FwdFlowToGateway(gatewayIPs, gatewayConfig.MAC, cookie.Default)...)
 	}
+	if runtime.IsWindowsPlatform() && c.encapMode.SupportsNoEncap() {
+		if gatewayConfig.IPv4 != nil {
+			flows = append(flows, c.l3FwdFlowRouteToWindowsGW(gatewayConfig.MAC, gatewayConfig.IPv4, cookie.Default)...)
```

Comment:
Can we leverage client_windows.go to avoid having platform specific function in client.go?

Reply:
Yes, updated.